web-ui/src/login/login.css

```diff
@@ -16,130 +16,47 @@
  */
```

Comment:
Ok. We don't need to do it in this story. But maybe we should open an issue for that?